presto-main/src/main/java/com/facebook/presto/cost/JoinStatsRule.java

```diff
@@ -49,12 +49,21 @@
         implements ComposableStatsCalculator.Rule
 {
     private static final Pattern PATTERN = Pattern.typeOf(JoinNode.class);
+    private static final double DEFAULT_UNMATCHED_ANTI_JOIN_NDVS_COEFFICIENT = 0.5;
```

Comment:
This is not for unknown values. This is for know columns, but we have to assume that not every NDV (from join side with less NDVs) matches "the other side". I tested and 0.5 causes slight overestimation (especially in q78) in `left joins`, which is probably good. I am not sure thought that the value is the best possible. If someone figures that some other value is better that is fine for me.
Lower value would increase anti join result set.
Greater values would decrease anti join result set.